### PR TITLE
Added Flygone60 Rev4

### PR DESCRIPTION
**Pull request by CheeseL0ver**
## Description

- Added files to match 4th revision of the PCB
- Updated the README to default to rev4

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/flygone60/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+![Flygone60](https://imgur.com/a/mZV0Dpi)
+
+A 60% that takes arrow keys seriously!
+
+* Keyboard Maintainer: [ShandonCodes](https://github.com/ShandonCodes)
+* Hardware Supported: Flygone60
+* Hardware Availability: ![Kit](https://www.etsy.com/listing/1025415290/flygone60-keyboard-kit?ga_order=most_relevant&ga_search_type=all&ga_view_type=gallery&ga_search_query=flygone60&ref=sr_gallery-1-13&organic_search_click=1&frs=1)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/flygone60/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/flygone60/rev4/rules.mk [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+# BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/flygone60/rev4/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/flygone60/rev4/rules.mk [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+# LAYOUTS = 60_ansi
```
Comment (drashna) [inline keyboards/flygone60/rev4/rules.mk:24]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/flygone60/rev4/rules.mk [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# Build Options
+#   change yes to no to disable
+#
+# BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/flygone60/rev4/rules.mk:16]:
```suggestion
```

File: keyboards/flygone60/rev4/info.json [hunk 5]
```diff
@@ -0,0 +1,151 @@
+{
+    "keyboard_name": "Flygone60 Rev4",
+    "url": "https://qmk.fm/keyboards/",
+    "maintainer": "ShandonCodes",
+    "width": 15,
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/flygone60/rev4/info.json:6]:
```suggestion
```

File: keyboards/flygone60/readme.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# flygone60
+
+![Flygone60](https://imgur.com/a/mZV0Dpi)
```
Comment (fauxpark) [inline keyboards/flygone60/readme.md:3]:
```suggestion
![Flygone60](https://i.imgur.com/x63gMG2h.jpeg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**